src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/AmendRestitchBundleCallbackHandler.java

```diff
@@ -95,4 +96,18 @@ private SubmittedCallbackResponse buildConfirmation(CallbackParams callbackParam
             .confirmationBody("Placeholder")
             .build();
     }
+
+    private CallbackResponse amendRestitchBundle(CallbackParams callbackParams) {
+
+        CaseData caseData = callbackParams.getCaseData();
+        CaseData.CaseDataBuilder dataBuilder = caseData.toBuilder();
+
+        if (caseData.getBundleError() == null || caseData.getBundleError().equals(YesOrNo.NO)) {
```

Comment:
I think you can remove the if here Kannan, Miguel's ticket should prevent submission upon bundle error.
So you shouldn't need to check for it.

Reply:
@m-meulendijks-v1 Removed